h/config.py

```diff
@@ -117,7 +117,7 @@ def configure(environ=None, settings=None):
 
     # Add ES logging filter to filter out ReadTimeout warnings
     es_logger = logging.getLogger('elasticsearch')
-    es_logger.addFilter(ExceptionFilter((("ReadTimeout", "WARNING"),)))
+    es_logger.addFilter(ExceptionFilter((("ReadTimeoutError", "WARNING"),)))
```

Comment:
Ah this is where are ES timeouts stopped being filtered out so they didn't go to sentry.